add osmooc speedblog (#390)
open-life-science/open-life-science.github.io: _posts/2022-07-18-ols5-speedblog-open-science-mooc.md

---
layout: post
title: "OLS-5 speedblog: Open Science MOOC, TU Delft"
authors:
- AleCandian
- lwbwalma
image: /images/2022-07-18-osmooc-logo.png
---

> ## Project background

Reviewer: ```suggestion
```

> The Delft University of Technology (TU Delft) in the Netherlands runs once a year a Massive Online Open Course (MOOC) on Open Science on the edX platform => [Open Science: Share your research with the world](https://learning.edx.org/course/course-v1:DelftX+OS101x+2T2022/home).  Our primary target is PhD candidates but the MOOC is meant also for MSc students, researchers, support staff, policy makers that want to learn the basic principles of and motivations behind Open Science.  

Reviewer: ```suggestion
The Delft University of Technology (TU Delft) in the Netherlands runs once a year a Massive Online Open Course (MOOC) on Open Science on the edX platform: [Open Science: Share your research with the world](https://learning.edx.org/course/course-v1:DelftX+OS101x+2T2022/home).

Our primary target is PhD candidates but the MOOC is meant also for MSc students, researchers, support staff, policy makers that want to learn the basic principles of and motivations behind Open Science.  
```

The OSMOOC has been running for multiple years with a decreasing number of enrollments and varying levels of engagement. We have high dropout rates and we do not know if the participants practice Open Science later on. Our project goal was to improve the engagement by creating a community around the MOOC participants.

![Logo of the Open Science MOOC](/images/2022-07-18-osmooc-logo.png)![titles of the six modules contained in the course: Introduction to Open Science, Open Data, Open Access Publishing, Visibility as a Researcher, FAIR Software, and Citizen Science](/images/2022-07-18-osmooc-modules.png)

_Figure1: Logo of the Open Science MOOC and titles of the six modules contained in the course. Image by TU Delft and licensed under [CC-BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) International License._

## Expectations from this program

> Since we had both just started in our jobs as open science teachers, the first thing we hoped to learn was more about the different themes within Open Science. We also hoped that through our participation in the OLS program, we would learn more ways to engage our MOOC’s participants and build a solid community, as well as workflows and processes to make our own work more open.

Reviewer: ```suggestion
Since we had both just started in our jobs as open science teachers, the first thing we hoped to learn was more about the different themes within Open Science. We also hoped that through our participation in the OLS program, we would learn more ways to engage our MOOC's participants and build a solid community, as well as workflows and processes to make our own work more open.
```

A final and eye-opening understanding was that we could improve how the MOOC team works together, implementing open working practices such as contributors guidelines. Our team is made of people coming from different departments within TU Delft and working on the MOOC only for a part of the year.  We talked with Emma Karoune  who gave us great practical advice both on community building activities for the MOOC team and on implementing governance structure to improve communication, task divisions and responsibility.

For me (Alessandra) learning how to use Github was perfect since I had it in my bucket list for a long time but never really found the time or motivation for it. For me (Lisanne) it was great to engage with fellow open scientists using Slack, LinkedIn and Twitter. I always wanted to start using social media for my work, and now I had a reason to.

## The main goals achieved in this project

### Engaging content

When possible, we updated the course content making it more relevant and engaging. We expanded the historical introduction to the Open Science Movement, introducing more diverse elements, such as women or people outside Europe.

> ![Historial illuminated manuscripts on parchment, with the title "Open Science: The beginning"](/images/2022-07-18-tudelft-history.png)

Reviewer: To change the width, you can use `{: width="20%" }` write after the image, e.g.: `![Historial illuminated manuscripts on parchment, with the title "Open Science: The beginning"](/images/2022-07-18-tudelft-history.png){: width="20%" }`

Author: Thanks!

_Figure 2: One of the slides used in the historical introduction video about the Open Science Movement. Material created by TU Delft and licensed under CC-BY-NC-SA 4.0 International License._

We also experimented with additional forms of feedback. We recorded a podcast where the module managers of FAIR software, research visibility and Citizen Science discussed the submitted assignments as well as overlapping themes in the different subjects, giving a more holistic view of the course.  We also recorded a talkshow, where representatives of our six modules came together to discuss advantages, challenges and future of the different module themes.

We also found new ways to promote the MOOC, reaching additional communities via social media (Twitter, LinkedIn, university newsletters) and mailing list.

![Twitter thread - linked in reference section for full text](/images/2022-07-18-twitter.png)

_Figure 3: Screenshot of Twitter threads created during the OSMOOC to engage on social media sharing additional resources and reflections. Links of the threads in the Reference Section._

> ## The initial steps

Reviewer: ```suggestion
## The initial steps

```

Our initial steps included a lot of brainstorming and making our goals clearer and achievable within the duration of the OLS program. Our mentor Patricia was very encouraging and supportive and created an environment where we could be ourselves and brainstorm freely, while at the same time giving us additional resources and asking us questions.
The result of our brainstorming sessions was to update as much MOOC content as we could for the 2022 run of the MOOC (our MVP), and to use the tools provided by the OLS program to reflect on our project management practices and create a plan to implement changes in our practices for next year’s MOOC run.

## What elements helped you get there?

> The MOOC’s course team showed enthusiasm to improve the content, such as including a podcast and feedback videos, and this motivated us. The biweekly sessions with our mentor also really helped us to focus and reflect. We were lucky to get many helpful tips from the OLS community (for example about historical content, ways to promote the MOOC, or making courses more engaging). Finally, the OLS program materials and sessions were insightful and tools such as ‘personas and pathways’ and the ‘project development plan’ help us improve and reflect on our MOOC and our plans for next year.

Reviewer: ```suggestion
The MOOC's course team showed enthusiasm to improve the content, such as including a podcast and feedback videos, and this motivated us. The biweekly sessions with our mentor also really helped us to focus and reflect. We were lucky to get many helpful tips from the OLS community (for example about historical content, ways to promote the MOOC, or making courses more engaging). Finally, the OLS program materials and sessions were insightful and tools such as ‘personas and pathways’ and the ‘project development plan’ help us improve and reflect on our MOOC and our plans for next year.
```

## Next steps

### Our immediate next step is to…

- Work to become a better and more open team. We started having lunch together to celebrate the end of the run. The first step toward this direction is a meeting that we are going to schedule by the end of summer to present the project development plan we created during the OLS course and get feedback on it.
- Following this meeting we want to draw a clear governance structure and governance framework for all contributors for next year’s run.

## Longer term tasks

> - Implement the new governance structure and framework.

Reviewer: ```suggestion

- Implement the new governance structure and framework.
```

> - Review how the MOOC went, look at the participants’ feedback and think about further improvements to the MOOC’s content and community building.

Reviewer: ```suggestion
- Review how the MOOC went, look at the participants' feedback and think about further improvements to the MOOC's content and community building.
```

## Staying connected

> - Stay in touch with the OLS community through Slack and become a mentor for the next cohort.

Reviewer: ```suggestion

- Stay in touch with the OLS community through Slack and become a mentor for the next cohort.
```

## Special mentions and acknowledgements

> - Thanks to the great OLS community especially the project leads Yo, Malvika, Bérénice and Emmy.

Reviewer: ```suggestion

- Thanks to the great OLS community especially the project leads Yo, Malvika, Bérénice and Emmy.
```

- Special thanks to Emmy for bringing this mentoring program to our attention and encouraging us to enroll
- Thanks to our mentor Patricia for directing our crazy brainstorming sessions, giving great advice and for being very nice :)

> - Thanks to Ismael-KG for giving us food for thought and ideas for our Open Science introduction thanks to his amazing  “Timeline”; to Frederique Belliard to discuss our journey in OLS-5 and Evelyn Greeves for feedback at different stages.

Reviewer: ```suggestion
- Thanks to Ismael-KG for giving us food for thought and ideas for our Open Science introduction thanks to his amazing  "Timeline", to Frederique Belliard to discuss our journey in OLS-5 and Evelyn Greeves for feedback at different stages.
```

> ## References
> @donnainfiorino (Alessandra Candian) “What is Open Science? …”, Twitter, 21 May 2022, https://twitter.com/donnainfiorino/status/1527919009395613696
> 
> @donnainfiorino (Alessandra Candian) “The 2nd week of #OSMOOC2022 is all about #OpenData. …”, Twitter, 31 May 2022, https://twitter.com/donnainfiorino/status/1531644206325895170
> 
> @donnainfiorino (Alessandra Candian) “The third week of the @tudelftlibrary #OSMOOC2022 has just come to an end and the topic was #OpenAccess. …”, Twitter, 8 June 2022, https://twitter.com/donnainfiorino/status/1534654326425149444
> 
> @donnainfiorino (Alessandra Candian) “During the last three weeks of the @tudelftlibrary @TUDelftOnline #OSMOOC2022 … “, Twitter, 24 June 2022, https://twitter.com/donnainfiorino/status/1540409251054714886

Reviewer: ```suggestion
## References

- [@donnainfiorino (Alessandra Candian) "What is Open Science? …", Twitter, 21 May 2022](https://twitter.com/donnainfiorino/status/1527919009395613696)
- [@donnainfiorino (Alessandra Candian) "The 2nd week of #OSMOOC2022 is all about #OpenData. …", Twitter, 31 May 2022](https://twitter.com/donnainfiorino/status/1531644206325895170)
- [@donnainfiorino (Alessandra Candian) "The third week of the @tudelftlibrary #OSMOOC2022 has just come to an end and the topic was #OpenAccess. …", Twitter, 8 June 2022](https://twitter.com/donnainfiorino/status/1534654326425149444)
- [@donnainfiorino (Alessandra Candian) "During the last three weeks of the @tudelftlibrary @TUDelftOnline #OSMOOC2022 … ", Twitter, 24 June 2022](https://twitter.com/donnainfiorino/status/1540409251054714886)
```

Reviewer: We could also embed the twitter threads in the post directly
